src/ql/utils/vcd.cc

```diff
@@ -83,7 +83,7 @@ void Vcd::change(Int var, Int timestamp, const Str &value)
 }
 
 
-void Vcd::change(Int var, Int timestamp, Int value) {
```

Comment:
I understand that the variable was not used, however here the variable names are clearly useful to know what the intent of the code was... or would have been...
This class is not inherited, so here is the only way to know what the code was...

This code has no owner anymore unfortunately. I would suggest not touching it.

Reply:
This is a good comment. I should have, in general, just commented out the variable names.

I've put the variable names back in again for this particular case. They're commented out though to avoid compilation warnings/errors.